batch encoder.bat

```diff
@@ -38,7 +38,7 @@ if /i "%1"=="--updated-from" (
 	set "UpdateVersion=%ver:~1%"
 	pause
 	
-	set complexcommand=powershell -Command "$x = Get-Content _assetlist.json -Raw; $k = $x | Select-String -Pattern '(?s)url(((?^^^!url).)*?)batch\.encoder'; $g = $k.Matches.Value | Select-String -Pattern '""[^"""]+?""",'; $g.Matches.Value"
+	set complexcommand=powershell -Command "$x = Get-Content _assetlist.json -Raw; $k = $x | Select-String -Pattern '(?s)url(((?^^^!url).)*?)batch\.encoder'; $k.Matches.Value; $g = $k.Matches.Value | Select-String -Pattern '^""[^^^^"""]+?^""",'; $g.Matches.Value"
```

Comment:
Escaping both regexes so CMD _and_ PowerShell respected it was a huge challenge. Wouldn't wish it on my worst enemy.

| Bare Regex | PowerShell Working | CMD + Powershell Working |
|--------|--------|--------|
| `url(((?!url).)*?)batch\.encoder` | `(?s)url(((?!url).)*?)batch\.encoder` | `(?s)url(((?^^^!url).)*?)batch\.encoder` |
| `"[^"]+?",` | <code>\`"[^\`"]+?\`",</code> | `^""[^^^^"""]+?^""",` |